docs/application/toc_all.md

```diff
@@ -83,6 +83,7 @@
 #### Components
 ##### [Overview](/application/dotnet/guides/nui/ui-components.md)
 ##### [Styling UI Components](/application/dotnet/guides/nui/styling-controls-with-JSON.md)
+##### [Styling UI Components with Xaml](/application/dotnet/guides/nui/styling-controls-with-xaml.md)
```

Comment:
I put styling themes in components, because another styling method (using JSON) is described here. It depends on you I think. For me it was better to keep information about components in components section.